util/getUniqueCurrentProjects.js

```diff
@@ -0,0 +1,11 @@
+export default function getUniqueCurrentProjects(teamMembers) {
+  const projectsList = Object.keys(teamMembers)
+    .map((key) => teamMembers[key].currentProjects)
+    .flat();
+
+  const projectNamesList = projectsList.map((project) => project.project);
+
+  const sortedUniqueProjects = [...new Set(projectNamesList)].sort();
```

Comment:
Very nice! We could split this up into two steps to separate the getting-unique-values part from the sorting part, but I don't feel super strongly either way.

Reply:
@theogainey decide this for us

Reply:
I think it would be best to spilt it up. Let's say someone decides it would be a good idea to sort the projects a different way, having a clear sort step would make it easier for that someone to implement that.

Reply:
🏆